pandas/io/tests/test_pickle.py

```diff
@@ -148,6 +149,21 @@ def _validate_timeseries(self, pickled, current):
         self.assertEqual(pickled.index.freq.normalize, False)
         self.assert_numpy_array_equal(pickled > 0, current > 0)
 
+    def _validate_frequency(self, pickled):
```

Comment:
@jreback Older pickles are tested in here. Also, I've added my pickle taken in v0.15.2.